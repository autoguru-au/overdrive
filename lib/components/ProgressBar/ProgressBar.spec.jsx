import { render } from '@testing-library/react';
import * as React from 'react';

import { ProgressBar } from '.';

describe('<ProgressBar />', () => {
	it('should not throw', () => {
		expect(() => render(<ProgressBar percentage={0.1} />)).not.toThrow();
	});

	it('should match the snapshot', () => {
		expect(
			render(<ProgressBar percentage={0.5} />).container.firstChild,
		).toMatchSnapshot();
	});
});
