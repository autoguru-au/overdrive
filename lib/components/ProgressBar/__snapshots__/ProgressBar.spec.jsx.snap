// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProgressBar /> should match the snapshot 1`] = `
<div
  class="root"
>
  <div
    class="barBackdrop"
  >
    <div
      class="barCurrentStatus green"
      style="width: 0%;"
    />
  </div>
</div>
`;
